src/lwc/rd2EntryForm/rd2EntryForm.js

```diff
@@ -463,10 +472,17 @@ export default class rd2EntryForm extends LightningElement {
         const period = getFieldValue(this.record, FIELD_INSTALLMENT_PERIOD);
         const campaignId = getFieldValue(this.record, FIELD_CAMPAIGN);
 
-        return amount !== Number(allFields[FIELD_AMOUNT.fieldApiName])
-            || frequency !== Number(allFields[FIELD_INSTALLMENT_FREQUENCY.fieldApiName])
-            || period !== allFields[FIELD_INSTALLMENT_PERIOD.fieldApiName]
-            || campaignId !== allFields[FIELD_CAMPAIGN.fieldApiName];
+
+        const hasFrequencyField = allFields.hasOwnProperty(FIELD_INSTALLMENT_FREQUENCY.fieldApiName);
```

Comment:
Interesting, was there a situation that this field wasn't available? If so, my bad!

Reply:
The frequency field is only present when we're using Advanced scheduling.